### PR TITLE
Journal week 5 from master

### PR DESCRIPTION
**Pull request by jmcreasman**

### CODE REVIEW

**Comment by jeffworley** [comment]
**Very** solid start!  I am especially interested to explore your resource.  Anything that can use **BATMAN** and make _lodash_ more understandable is priceless. :+1:

**Comment by boshnivolo** [comment]
You have some really fluid writing in your journal! It was a good reflective piece :+1:

**Comment by boshnivolo** [comment]
Great tutorial! I especially enjoyed the code fenced section displaying what all the different replacements for `<div>`s would look like together. Can't wait to see what it'll turn into! :+1:

**Comment by boshnivolo** [comment]
That's a handy resource you got there! Thanks for sharing! :+1:

**Comment by ShayHall** [comment]
Pretty good start.  I laughed at the part of the reflective journal about the logic problem.  I was just talking about that the other day.  In your tutorial, two things I was looking for that I didn't see: ["semantic HTML"](https://en.wikipedia.org/wiki/Semantic_HTML) and "box-sizing".  Perhaps you can work those concepts in in the final draft.  :+1:

**Comment by boshnivolo** [comment]
I like the change in format you did for your journal! :+1: 
And Batman is always the best resource :+1: 
And the tutorial helps me strengthen my fundamentals :+1:

**Comment by Ricecoder** [comment]
I agree with you about logic in this field. It plays such a big role in how we build our sites. :+1: 
Your resource is great but then again it has Batman... :+1:
I really like the picture you linked to in your tutorial. Very helpful! :+1:

File: journal-week-5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+##Do You Even Logic?
+During my interview process here I had my one on one interview with Kelly. Near the end she asked me a question that I was not expecting. It was a logic based puzzle and I don’t exactly remember what the question was or what answer I gave (other than that is was way more complicated than it needed to be). What I do remember was distinctly wondering why logic would be important or relevant in programing. If six weeks into the Iron Yard has taught me anything it is that logical thinking is a VERY important skill to have.
```
Comment (kellymurray) [inline journal-week-5.md:2]: :+1:

File: resource-week-5.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+http://colintoh.com/blog/lodash-10-javascript-utility-functions-stop-rewriting
+
+Well first of all half the reason I picked sharing this article was because it used Batman as an example and Batman’s just cool. The other half of that equation was that it takes a new and complicated subject for us (Lodash) and easily demonstrates 10 really useful reasons to use it. Granted we probably won’t be using a lot of these at our level of learning just yet. But I found it helpful in wrapping my mind around several js concepts I hadn’t even thought of yet. And yeah did I mention Batman?
```
Comment (kellymurray) [inline resource-week-5.md:3]: :+1:

File: tutorial-week-5.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+##Website Layout Using HTML5
```
Comment (kellymurray) [inline tutorial-week-5.md:1]: You might call this _Semantic_ HTML. `<div>`s are part of HTML5 - just not _semantic_.

**Pull request merged**